nucliadb/src/nucliadb/export_import/utils.py

```diff
@@ -40,7 +40,7 @@
 from nucliadb_utils.const import Streams
 from nucliadb_utils.transaction import MaxTransactionSizeExceededError
 
-BinaryStream = AsyncGenerator[bytes, None]
+BinaryStream = Union[AsyncGenerator[bytes, None], AsyncIterator[bytes]]
```

Comment:
A generator is a kind of iterator, so you can just do this (+ fix imports):

```suggestion
BinaryStream = AsyncIterator[bytes]
```